include/TcpMainService.h

```diff
@@ -22,6 +22,8 @@ class TcpMainService: public Command
 private:
 	Command * getCommand(size_t opcode, int socketFd);
 	void tcpServiceLoop(void);
+
+	const unsigned port = 8888;
```

Comment:
we should have one port's variable - in NetMainThread: static const unsigned port